tests/ert_tests/ert3/data/test_record.py

```diff
@@ -35,18 +35,53 @@ def test_valid_record(data):
         assert data[idx] == record.data[idx]
 
 
+@pytest.mark.parametrize(
+    "data",
+    (
+        [b""],
+        [b"abcde"],
```

Comment:
Can we have at least one list with multiple elements all the time it is explicitly supported by the blob record definition?

Reply:
It is indeed supported, by the definition, if not by the rest of the code. I have added some test entries accordingly.